src/ResourceManager/Network/Commands.Network.Test/ScenarioTests/VirtualNetworkGatewayTests.cs

```diff
@@ -51,5 +51,11 @@ public void VirtualNetworkGatewayActiveActiveFeatureTest()
         {
             NetworkResourcesController.NewInstance.RunPsTest("Test-VirtualNetworkGatewayActiveActiveFeatureOperations");
         }
+
+        [Fact(Skip = "Will record later")]
```

Comment:
Why are you not recording now?  If this functionality is not available, then this PR cannot be emrged.  If you're sure the functionality is working in production, then you must file an issue in the repo to record thes tests in the next sprint and reference it here